.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint
```

Comment:
I'm not very used to gh-actions and tox, but isn't flake8 already being tested in `tests.yml` below?